apps/arena/lib/arena/game_tracker.ex

```diff
@@ -0,0 +1,118 @@
+defmodule Arena.GameTracker do
+  @moduledoc """
+  This module is in charge of tracking the different game changes that we consider
+  datapoints and use for quests, analytics, etc.
+
+  `GameTracker` will behave similar to a metrics collector, but with a push model rather than pull.
+  Games will push the data to it and every X time it will push the data to Gateway for storing and updating quests
```

Comment:
Every X time? I think every time a match ends, it sends the data to Gateway, which is desirable.

Reply:
Yep, fixed the words. They are from before changing to only the final update